README.md

```diff
@@ -1 +1,162 @@
\ No newline at end of file
+<a name="readme-top"></a>
+
+  <h3><b>Essohanam Portfolio README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [xGen Conference] <a name="about-project"></a>
+
+> Hello , This is my first project and its a conference website.
+
+**[CC Environment Global Conference]** "It's a great website with well-defined sections for home and about pages, and it's responsive on screens below 768px as well as on desktop versions." 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>HTML</summary>
+</details>
+
+<details>
+  <summary>CSS</summary>
+</details>
+<details>
+  <summary>JavaScript</summary>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Dynamic speakers section]**
+- **[Menu clickable]**
+- **[Hover Buttons]**
+- **[Responsive desplay]**
+- **[colorful texts]**
+- **[About ]**
+- **[Icons ]**
+- **[Validation Form]**
+- **[Aligned items]**
+- **[Desktop screens supported]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://abbassarwar.github.io/XGen-Conference/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+
+### Prerequisites
+
+In order to run this project you need: 
+
+## Recommendation
+1. A good Computer
+2. Having proper knowledge about Css and Html **necessary**
+
+* Live check : https://abbassarwar.github.io/XGen-Conference/
+* git clone https://github.com/AbbasSarwar/XGen-Conference.git
+* cd XGen-Conference
+* Go live through in VS code for further.
+* Here we Go....
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@GhostEsso](https://github.com/GhostEsso)
+- Twitter: [@TambanaEssohana](https://twitter.com/TambanaEssohana)
+- LinkedIn: [Tambana Essohanam](https://www.linkedin.com/in/essohanam-tambana-62aa081a1/)
+- Credit: [Cindy Shin](https://www.behance.net/adagio07)
```

Comment:
- [ ] Good job, acknowledging Cindy here nut it is more noticeable and gives the appropriate credit in the acknowledgment section. In addition, I think it will be more clear if you add a little bit of details in addition to the name. E.g: `This project was inspired and originally designed by [Cindy Shin](https://www.behance.net/adagio07) on behance` - Something descriptive enough to ensure clarity. :hammer_and_wrench: